toolkits/springernature/packages/springernature-submission-header/scss/50-components/core.scss

```diff
@@ -0,0 +1,24 @@
+.c-submission-details-header {
+	font-size: 12px;
+	font-size: 0.75em;
+	margin: 0 0 20px 0;
```

Comment:
some of these values such as `20px` are repeated in your component so could benefit from being made in to scss variables and stored in a 10-settings file.

Reply:
also, do you have a set of spacing values defined in your brand context as scss variables? as they could be referenced here

Reply:
I am not sure I see the value in that, they are not really linked, and unlikely to be changed.

Reply:
Ah ok, we have a map of spacing values in our brand context that all components reference to enforce spacing consistency on our sites. I thought perhaps here the 20px values might be a spacing value you often use. But if they are unlinked and you don't have a standard set of spacing values anywhere then that's fair enough.

Reply:
It's a good idea though, and something on the radar.